modin/sql/connection.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from ..pandas import Series, DataFrame
```

Comment:
Will the `ray.init` call run if DataFrame is imported relatively?

Reply:
It does run, Yes

Reply:
Perfect, thanks for confirming.